types/index.ts

```diff
@@ -318,7 +324,7 @@ export type TechnologyType = {
   buttonBackground: string;
 };
 
-export type ProjectDetailsTopType = {
+export type TopContentType = {
```

Comment:
I also want to add one thing if you don't mind, @AlexDjangoX. Be sure to use descriptive names in your code.  I was reviewing this to see how we could group these together in a better, more organized approach, but I don't know what 'TopContentType' means or where to begin looking for it. (Pretend I don't have 'alt f'...) Being descriptive improves the readability of the code. Just a suggestion...